build/config/win/BUILD.gn

```diff
@@ -32,6 +32,11 @@ config("winver") {
   ]
 }
 
+# When OS target is WinUWP use WINUWP as the preprocessor flag to conditionalize.
+config("flutter_windows_winuwp_defs") {
+  defines = [ "WINUWP" ]
```

Comment:
Shouldn't we follow [the Chromium approach](https://source.chromium.org/chromium/chromium/src/+/master:build/config/win/BUILD.gn;l=223?q=defines%20winuwp&ss=chromium) and put this in the `sdk` config above behind an OS check, so that it's automatically applied to all the targets that are getting the other defines?

Reply:
Great idea.  Addressed.